documentation/writing_plans.md

```diff
@@ -224,6 +224,80 @@ plan mymodule::myplan {
 }
 ```
 
+## Debugging plans
+
+By default, Bolt does not print verbose logs for each plan execution to stdout.
+However, you can use one of the following methods to investigate a plan
+execution:
+- Each time you run a Bolt command, Bolt prints a debug log to a
+  `bolt-debug.log` file in the root of your project directory.
+- You can use the `--verbose` CLI option for verbose logging to stdout.
+- You can print a parameter to stdout using the `out::message` function. 
```

Comment:
```suggestion
- You can print to stdout using the `out::message` function. 
```
Only since 'parameter' has specific meaning in plans, and you can print anything with out::message not just plan parameters